package/yast2-registration.spec

```diff
@@ -58,6 +58,13 @@ BuildRequires:  yast2-update >= 3.1.19
 
 BuildArch:      noarch
 
+# FIXME: it seems can we cannot move it to macros.yast, the yast-rake-ci is not
+# installed into the chroot, the build fails...
```

Comment:
Yes, I meant moving these lines to `macros.yast`.

Unfortunately it does not work, it seems that `osc` does not evaluate the spec file correctly when preparing the build root. It does not install the `rubygem-rake-ci` package into the build root. Later `rpmbuild` fails with missing BuildRequires error... :worried: